Makefile

```diff
@@ -54,7 +53,7 @@ INC_DIRS += $(UT_CORE_DIR)/src
 
 SRC_DIRS += $(UT_CORE_DIR)/src
 
-XLDFLAGS += -L $(UT_CONTROL)/lib -lut_control
+XLDFLAGS += -L $(UT_CONTROL)/lib/$(TARGET) -lut_control
```

Comment:
ut-control should have copied it's binaries to the ut-core lib directory already as part of it's build requirement. so all libraries come from a common directory.

That way you can verify that the client with also work